ingestion/src/metadata/ingestion/source/database/snowflake/metadata.py

```diff
@@ -322,7 +322,6 @@ def query_table_names_and_types(
             TableNameAndType(name=table_name)
             for table_name in self.inspector.get_table_names(
                 schema=schema_name,
-                include_temp_tables=self.service_connection.includeTempTables,
```

Comment:
hi qq, why is this not needed here anymore? I might be missing some context on how we were actually defining "TMP" tables, but I see that this was used to eventually run this query https://github.com/open-metadata/OpenMetadata/blob/main/ingestion/src/metadata/ingestion/source/database/snowflake/queries.py#L48

My point is, if this is indeed not needed, then we should clean the argument in the function etc. But i'd like to understand how we are picking all the different ingredients. A couple of comments / docstrings would help.

Thanks

Reply:
and btw, thanks for taking the time to contribute. Appreciate it

Reply:
Hi @pmbrull , In snowflake transient tables are considered as temporary tables, earlier we are adding these temp tables in regular_tables because of that in UI we were getting table type of transient table as "Regular" but now you can see I created a new variable "Transient_table" there I am passing the list of temp tables with its table type as Transient.

Reply:
I hope I answer your question :) please feel free to ask me if you have any doubts, also I will add the comments in code too .
Thank you 😊

Reply:
aaaah ok I see, thanks. Sorry I misread the code before.

I see however that snowflake differentiates between Transient and Temporary https://docs.snowflake.com/en/user-guide/tables-temp-transient, is this a difference we can pick up / are we picking up currently?

Thanks

Reply:
I checked the document , I see now its not extracting the temp tables , l am looking into it and will try to cover all scenarios.

Thanks

Reply:
Hey @pmbrull @vanshika18, as far as I remember while developing this feature to extract the transiant tables we noticed that the temporary tables that Snowflake supports are session specific, once the session is killed the table would not exist, so we can not extract those to OM because while extracting tables to OM we would have to open a new session which would not contain that temporary table. (at least this is what I remember) also if that is the case we should maybe name the variables a bit better here 😅

Reply:
thanks, let's extract whatever is available. IMO the important part to me here is that these decisions - even if led by design - need to be reflected in the docs. Otherwise even us we'll forget since they are not trivial topics